spec/features/workflow/publish_without_selection_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.feature "Attempting to publish without confirming a review status" do
+  scenario do
+    given_there_is_an_edition_in_draft
+    when_i_visit_the_summary_page
+    and_try_and_publish_without_selecting_a_review_status
```

Comment:
Also good to synchronise the terminology here (`status` vs. `state`).

Reply:
Yep nice spot!